{"error" : "anomaly: Invalid_argument(\"Sys.signal: unavailable signal\").\n" , "internal" : true , "results" : [] }